tests/functional/adapter/test_query_comments.py

```diff
@@ -8,25 +10,39 @@
 )
 
 
+# TODO: below tests could be simplified to just
```

Comment:
have you checked for any open issues/PRs related to this issue?

Reply:
Yes I checked, but I didn't find any such issue. I will open issue in dbt-core myself.